src/midi-router.js

```diff
@@ -9,7 +9,7 @@
 "use strict";
 
 var fluid = require("infusion"),
-    flock = fluid.require("flocking");
+    flock = fluid.require("flocking"); // eslint-disable-line no-unused-vars
```

Comment:
You have to do this for global variables that are resolved through IoC.  They need to exist globally in order to be resolved.  It comes up a lot with IoC tests and node-jqunit, for example.